core/src/test/java/kafka/server/share/SharePartitionManagerTest.java

```diff
@@ -1221,6 +1224,36 @@ public void testReplicaManagerFetchShouldProceed() {
             any(), any(), any(ReplicaQuota.class), any());
     }
 
+    @Test
+    public void testCloseSharePartitionManager() throws Exception {
+        SharePartitionManager sharePartitionManager = SharePartitionManagerBuilder.builder()
+                .withShareGroupPersister(mock(Persister.class)).build();
```

Comment:
Why do we require a persister mock when we are not using the mock instance?

Reply:
you're right, we don't need to mock it. I have removed it.